src/NexusMods.Abstractions/CLI/RegisteredVerb.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace NexusMods.Abstractions.CLI;
+
+/// <summary>
+/// Represents an individual action, e.g. 'Analyze Game'
+/// </summary>
+public class RegisteredVerb
```

Comment:
This lives outside the IVerb, AVerb and implementations hierarchy but the comment is identical to IVerb.
Some clarification regarding their relationship could be useful in the comment.
If I read the code right, this something like a wrapper for verbs when registered in the DI system.